### ISSUE TITLE
Quick input missing required ARIA state

### ISSUE THREAD

**Issue by scnwwu** [microsoft/vscode#174317]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.75.1
- OS Version: Windows 10

Steps to Reproduce:

1. Open Command Palette
2. Inspect the quick input DOM

```
<input class="input" autocorrect="off" autocapitalize="off" spellcheck="false" type="text" wrap="off" aria-describedby="quickInput_message" placeholder="Type the name of a command to run." style="background-color: inherit; color: var(--vscode-input-foreground); width: calc(100% - 0px);" aria-label="Type the name of a command to run." role="combobox" aria-haspopup="true" aria-autocomplete="list" aria-activedescendant="list_id_11_0">
```

It has role `combobox` but without `aria-expanded`. `aria-expanded` is required for `combobox` role. Refers to
https://w3c.github.io/aria/#combobox

It's reported as a level A violation to WCAG [Criteria 4.1.2](https://www.w3.org/TR/WCAG21/#name-role-value) by accessibility scan.

Fix would probably look something like below
For file https://github.com/microsoft/vscode/blob/main/src/vs/platform/quickinput/browser/quickInput.ts
```
private setComboboxAccessibility(enabled: boolean) {
	if (enabled !== this.comboboxAccessibility) {
		const ui = this.getUI();
		this.comboboxAccessibility = enabled;
		if (this.comboboxAccessibility) {
			ui.inputBox.setAttribute('role', 'combobox');
			ui.inputBox.setAttribute('aria-haspopup', 'true');
			ui.inputBox.setAttribute('aria-expanded', ui.list.isDisplayed()); // <- add
			ui.inputBox.setAttribute('aria-autocomplete', 'list');
			ui.inputBox.setAttribute('aria-activedescendant', ui.list.getActiveDescendant() || '');
		} else {
			ui.inputBox.removeAttribute('role');
			ui.inputBox.removeAttribute('aria-haspopup');
			ui.inputBox.removeAttribute('aria-expanded'); // <- add
			ui.inputBox.removeAttribute('aria-autocomplete');
			ui.inputBox.removeAttribute('aria-activedescendant');
		}
	}
}
```

**Comment by TylerLeonhardt** [comment]
We no longer use combobox so this isn't needed anymore. See:
https://github.com/microsoft/vscode/issues/237324#issuecomment-2638610328
